services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -1047,13 +1047,17 @@ async def patch_project_node(
     if _node_patch_exclude_unset.get("label"):
         await dynamic_scheduler_api.update_projects_networks(app, project_id=project_id)
 
-    # 5. Updates project states for user, if inputs have been changed
-    if "inputs" in _node_patch_exclude_unset:
+    # 5. Updates project states for user, if inputs/outputs have been changed
+    if {"inputs", "outputs"} & _node_patch_exclude_unset.keys():
```

Comment:
`a & b` with `a,b` being sets, is an intersection?   I would use `a.intersects(b)`

Reply:
Yup, it's intersection https://docs.python.org/3.10/library/stdtypes.html#frozenset.intersection